ttl.py

```diff
@@ -0,0 +1,153 @@
+"""Module for TTL status management."""
+
+import asyncio
+import dataclasses
+import enum
+import time
+from typing import Any
+
+from artiq.coredevice.comm_moninj import CommMonInj, TTLOverride, TTLProbe
+
+from protocols import SortedQueue
+
+device_to_channel: dict[str, int] = {}
+channel_to_device: dict[int, str] = {}
+
+def map_device_channel(ttl_devices: list[str], device_db: dict[str, Any]):
+    """Maps TTL devices and channels.
+    
+    Args:
+        ttl_devices: See main.configs.
+        device_db: See main.device_db.
+    """
+    for device in ttl_devices:
+        channel = device_db[device]["arguments"]["channel"]
+        device_to_channel[device] = channel
+        channel_to_device[channel] = device
+
+
+@enum.unique
+class MonitorType(enum.Enum):
+    """Monitoring value type."""
+    PROBE = "probe"
+    LEVEL = "level"
+    OVERRIDE = "override"
+
+
+@dataclasses.dataclass
+class StatusType:
+    """Monitoring status type.
```

Comment:
I don't get the meaning of "monitoring status".
This is merely a combination of a channel and a monitor type.
Could you explain about the intuition?

Reply:
I got your point. Thank you for the explanation.